### DISCUSSION TITLE
Thicker or bold image text when converting from .eps to .tif

### DISCUSSION THREAD

**Discussion by juloboyle** [ImageMagick/ImageMagick#3274] [Help]
Using the below command to convert an .eps file to .tif, but the text in the tif appears much darker or bolder.  I tried the -weight parameter with no luck.  Any idea on why the text is appearing darker?

convert -depth 8 -density 300x300 -units PixelsPerInch -weight Thin -compress Group4 C:\Temp\IM\34.eps  C:\Temp\IM\34_WeightThin.tif

snapshot of .eps: 
![image](https://user-images.githubusercontent.com/79325264/108528444-6aadac80-72a1-11eb-8f31-4bb2a1df7aca.png)

snapshot of .tif:  
![image](https://user-images.githubusercontent.com/79325264/108528564-8c0e9880-72a1-11eb-9bd7-2cf8692d34d7.png)

Thank you for the help!

**Comment by snibgo** [comment]
IM delegates rasterization of EPS files to Ghostscript. "-verbose" may show the GS command. I don't think "-weight" will be passed on to GS.

A larger "-density" will reduce the horribleness of the edges, and removing the compression will also help.

**Reply by juloboyle** [reply]
Thanks for the -verbose tip.  -weight does not get passed to GS.  I increased density to 600 and it improved, however, I need to produce 300.

**Comment by fmw42** [comment]
Your original text is anti-aliased. When you compress to Group 4 it becomes binary and the anti-aliasing is lost.

**Comment by snibgo** [comment]
>  I increased density to 600 and it improved, however, I need to produce 300.

You can resize after the rasterization, for example:
```
convert -depth 8 -density 600x600 -units PixelsPerInch -weight Thin -compress Group4 C:\Temp\IM\34.eps -resize 50% C:\Temp\IM\34_WeightThin.tif
```
If the command is in a BAT script, you need to double the % sign: %%.

What version of IM do you use? If 7, I suggest you use "magick", not "convert".

Please link to your input 34.eps.

**Reply by juloboyle** [reply]
Thanks snibgo.  I'll try magick instead.  btw, I had to attach as .zip, as this control doesn't support .eps upload.
[34_test.zip](https://github.com/ImageMagick/ImageMagick/files/6022497/34_test.zip)

**Comment by snibgo** [comment]
Rasterizing at 450 dpi creates an output very similar to your "snapshot of .eps":
```
magick -density 450 34_test.eps x.png
```
![x](https://user-images.githubusercontent.com/33812368/108724143-eb2c1180-751c-11eb-9b62-bb28c8ee15cb.png)

Compressing with the Group4 method removes any grayness, so makes it look horrible.

**Reply by juloboyle** [reply]
I need to use compression.  I was experimenting with upconverting to 600 compressed, then back down to 300 and got a pretty decent result:

magick ^ -density 600 -units PixelsPerInch -depth 8 -compress Group4 -monochrome +antialias c:\temp\im\34.eps -write c:\temp\im\34_300c_onecommand.tif ^ -density 300 c:\temp\im\34_300c_onecommand.tif

Is there a way to avoid the "-write"?  can I do both the upconvert/downconvert commands in memory?

**Comment by snibgo** [comment]
You write to the same file twice. The second write will override the first, so you can remove "-write c:\temp\im\34_300c_onecommand.tif".

Your first "-density 600", before the EPS is raserized, has an effect on the image.

Your second "-density 300" has no effect on the image. It only changes metadata.

**Reply by juloboyle** [reply]
Got it.  Thank you very much for your help!
